lms/templates/vert_module.html

```diff
@@ -17,3 +17,28 @@ <h2 class="hd hd-2 unit-title">${unit_title}</h2>
   %endif
 % endfor
 </div>
+
+<script>
+  $(function(){
+    var divs = document.getElementsByClassName('vert-mod');
+    if(divs.length > 0) {
+      for (var i = 0; i < divs.length; i++) {
+          var anchors = divs[i].getElementsByTagName('a');
+          if(anchors.length > 0) {
+            for (var j = 0; j < anchors.length; j++) {
+              var hrefVal = anchors[j].href;
+              if(hrefVal && hrefVal != '') {
+                var fileName = hrefVal.split('/').pop();
+                var ext = fileName.split('.').pop();
+                if(ext === 'csv') {
```

Comment:
Can't we directly get the links that contain `csv` in the link & modify those instead iterating through the divs & anchors?
i.e 
```
$('a[href*="csv"]').each(function() {
    // code here
});
```